scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamMode.scala

```diff
@@ -74,16 +73,15 @@ object BeamSink extends Serializable {
 
   def textLine(path: String): BeamSink[String] =
     new BeamSink[String] {
-      override def write(pipeline: Pipeline, config: Config, pc: PCollection[_ <: String]): Unit =
+      override def write(pc: PCollection[_ <: String], config: Config): Unit =
         pc.asInstanceOf[PCollection[String]].apply(TextIO.write().to(path))
```

Comment:
nit: can we use our safer `widen` function here (general cast is more dangerous).

Reply:
Done